index.html

```diff
@@ -1 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
 
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="https://altur-ui.netlify.app/stylesheets/main.css">
+    <link rel="stylesheet" href="style.css">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <title>Altur Store</title>
+</head>
+
+<body>
+
+    <nav class="flex-center-se">
```

Comment:
It is not very clear with the class name as to what it does, is it `justify-content: center or space-evenly` or `align-items: center or space-evenly`. You can create classes like, `justify-content-sp-ev | sp-bw | center | start | end`